bentoml/_internal/bento/bento.py

```diff
@@ -318,6 +319,14 @@ def validate(self):
     def __str__(self):
         return f'Bento(tag="{self.tag}")'
 
+    @contextlib.contextmanager
+    def patch_context(self):
+        with BentoMLContainer.model_store.patch(self._model_store):
+            pwd = os.getcwd()
+            os.chdir(self.path)
+            yield
+            os.chdir(pwd)
+
```

Comment:
Do we want to raise if another bento has already patched context for model store?